anyway/views/news_flash/api.py

```diff
@@ -103,3 +105,16 @@ def single_news_flash(news_flash_id: int):
             json.dumps(news_flash_obj.serialize(), default=str), mimetype="application/json"
         )
     return Response(status=404)
+from sys import maxsize
+@user_optional
+def get_road_type(lat: float, lon: float):
+    offset = 0.00001
```

Comment:
How did you decide on the offset size (0.00001)?
In general - I suggest using geographical query instead of the query below -[see geom.intersects here](https://github.com/hasadna/anyway/blob/dev/anyway/models.py#L397)

In this case - I think that using both the geographical query and the road_number / street name together can be very powerful - just using one of them is not enought.
However - we can start with geographical query and improve it.